deploy/playbooks/ansible.cfg

```diff
@@ -0,0 +1,20 @@
+#
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c) 2020 The Authors.
+# Authors: Bin Liang  <@liangbin>
+#
+# Summary: Project specific Ansible configuration
+#
+# Paths defined in ansible.cfg are relative to the ansible.cfg file location
+[defaults]
+#inventory               = inventories/hosts
+roles_path              = roles
+vault_password_file     = inventories/vars/.vault_pass
+log_path                = /var/log/ansible.log
+host_key_checking       = False
+command_warnings        = False
+deprecation_warnings    = False
+display_skipped_hosts   = no
+retry_files_enabled     = False # Do not create them
+
```

Comment:
Since we require Python3 with ansible would it be a good idea to set
ansible_python_interpreter=/usr/bin/python3 here?

Reply:
I'm a little bit hesitated about this one, two concerns:
1. The path is always system specific so not good idea to put it in
2. I'm not sure if python 3 is absolutely required, we can try verify it later
what do you think?

Reply:
Resolved offline, Phu is ok with current approach